packages/mdc-textfield/_mixins.scss

```diff
@@ -250,15 +262,34 @@
     z-index: 2;
   }
 
-  // stylelint-disable plugin/selector-bem-pattern
-  .mdc-text-field__input:hover ~ .mdc-text-field__idle-outline {
-    border: 1px solid;
+  .mdc-notched-outline,
+  .mdc-notched-outline__idle {
+    @include mdc-text-field-outline-corner-radius($mdc-text-field-border-radius);
```

Comment:
calling a text-field mixin within the scope of `.mdc-notched-outline` seems backwards. Shouldn't we call a notched outline mixin here?

Reply:
good catch...that should be a notched-outline mixin